### PR TITLE
fix the carry on base type multiplication

### PR DESCRIPTION
**Pull request by greged93**
Fixes the carry on multiplication for the base type.

### CODE REVIEW

File: src/field/types/base.rs [hunk 1]
```diff
@@ -328,6 +329,33 @@ mod tests {
...
+                7771008069523909290,
+                0,
+            ],
+        };
+        assert_eq!(expected.value, result.value);
+    }
+
+    #[test]
+    fn test_value() {
+        dbg!(9775813591145556650u64.overflowing_add(9775813591145556649));
```
Comment (jobez) [inline src/field/types/base.rs]: lets remove debugging tests :)

**Review by jobez — CHANGES_REQUESTED** [review]
great work! small comment!

File: src/field/types/base.rs (hunk 1 above)
Comment (greged93) [reply src/field/types/base.rs]: oups

**Review by jobez — APPROVED** [review]
wahoo

**Pull request merged**